app/models/user.js

```diff
@@ -132,5 +133,9 @@ export default Model.extend({
     user = JSON.parse(this.get('sessionStorage').getItem('travis.user'));
     user[name.underscore()] = this.get(name);
     return this.get('sessionStorage').setItem('travis.user', JSON.stringify(user));
-  }
+  },
+
+  avatarUrl: function() {
+    return gravatarImage(this.get('email'), 36);
+  }.property()
```

Comment:
this should depend on `email`, so if `email` ever changes it will get recomputed. That's probably not really that important at the moment, because email won't rather change without reloading, but it will be more future proof.